knimin/lib/data_access.py

```diff
@@ -2415,6 +2415,126 @@ def set_deposited_ebi(self):
                  WHERE barcode IN %s"""
         self._con.execute(sql, [barcodes])
 
+    def create_study(self, study_id, title=None, alias=None, notes=None):
+        """Creates a study
+
+        Parameters
+        ----------
+        study_id : int
+            Positive if it is a Qiita study ID, negative if others
+        title : str
+        alias : str
+        notes : str
+
+        Raises
+        ------
+        ValueError
+            If there is error creating the study
+        """
+        with self._con.cursor() as cur:
+            cur.execute('BEGIN')
```

Comment:
I guess I understand your rationale. @wasade and @josenavas please make a decision which way to go. I am not capable of implementing the Qiita transaction object. So can anyone do it if we want to go this way?